ddl/rollingback.go

```diff
@@ -118,6 +118,42 @@ func rollingbackAddColumn(t *meta.Meta, job *model.Job) (ver int64, err error) {
 	return ver, errCancelledDDLJob
 }
 
+func rollingbackDropColumn(t *meta.Meta, job *model.Job) (ver int64, err error) {
```

Comment:
How about remove this function. There are too many redundant code. Use `JobStateRollingback`
to do rollback in `onDropColumn` function.